Gemfile

```diff
@@ -28,22 +28,14 @@ group :system_tests do
   gem 'beaker-puppet_install_helper', :require => false
 end
 
-is_x64 = Gem::Platform.local.cpu == 'x64'
-if is_x64
-  platform(:x64_mingw) do
-    gem "win32-dir", "~> 0.4.9", :require => false
-    gem "win32-process", "~> 0.7.4", :require => false
-    gem "win32-service", "~> 0.8.4", :require => false
-    gem "minitar", "~> 0.5.4", :require => false
-  end
-else
-  platform(:mingw) do
-    gem "win32-process", "~> 0.6.5", :require => false
-    gem "win32-service", "~> 0.7.2", :require => false
-    gem "minitar", "~> 0.5.4", :require => false
-    if RUBY_VERSION =~ /^1\./ then
-      gem "win32console", :require => false
-    end
+platforms :mswin, :mingw do
```

Comment:
In fact, nearly this exact configuration is up with puppetlabs-acl - https://github.com/puppetlabs/puppetlabs-acl/blob/master/Gemfile#L40-L54

If the above was dangerous or didn't work, well then I'm not sure that AppVeyor agrees as it is running on 3.4.3, 3.7.5 x86/x64 (needs updated to 3.8.1), and 4.0.0 x86/x64 (needs updated to 4.2.0)  https://github.com/puppetlabs/puppetlabs-acl/blob/master/appveyor.yml#L13-L22

https://ci.appveyor.com/project/puppetlabs/puppetlabs-acl (take a look at each matrix, it outputs Environment vars, bundler, and gemfile.lock).